RNUnifiedContacts/RNUnifiedContacts.swift

```diff
@@ -163,17 +163,94 @@ class RNUnifiedContacts: NSObject {
 
       callback([error.localizedDescription, NSNull()])
     }
-
   }
-
-
-
-
-
+  
+  @objc func createContact(contactData: NSDictionary, callback: (NSObject) -> () ) -> Void {
+    
+    let mutableContact = CNMutableContact()
+    let contactStore = CNContactStore()
+    let req = CNSaveRequest()
+    
+    // TODO:
+    // Extend method to handle more fields
+    //
+    //
+    mutableContact.givenName = contactData["givenName"] as! String
+    mutableContact.familyName = contactData["familyName"] as! String
+    mutableContact.organizationName = contactData["organizationName"] as! String
+    
+    for number in contactData["phoneNumbers"] as! NSArray {
+      let transformedNumber = transformNumber( number as! NSDictionary )
+      mutableContact.phoneNumbers.append( transformedNumber )
+    }
+    
+    for email in contactData["emailAddresses"] as! NSArray {
```

Comment:
If the collection is called `emailAddresses` then a singleton should be called `emailAddress`, _not_ `email`.